galleon-plugins/src/main/java/org/wildfly/galleon/plugin/PropertyReplacer.java

```diff
@@ -106,7 +107,12 @@ public static void copy(final Reader reader, Writer writer, PropertyResolver pro
                                 } else if (ch == ',') {
                                     state = DEFAULT;
                                 } else {
-                                    throw new IllegalStateException("Failed to resolve property: " + buf);
+                                    if(ignoreFailure) {
```

Comment:
small nitpick - the `ignoreFailure` is not really needed, that could just check if `failureReplacement  != null`?

Reply:
Thanks, fixed.